src/components/Corpus.jsx

```diff
@@ -3,17 +3,70 @@ import Radium from 'radium'
 
 import Styles from './Styles'
 import Strings from './Strings'
+import TitleCard from './TitleCard'
 
 @Radium
-export default class Word extends React.Component {
+export default class Corpus extends React.Component {
+  delete = () => {
```

Comment:
don't call this delete, it may cause problems. Name it something like deleteCoprus